src/conn/conn_compact.c

```diff
@@ -165,9 +415,23 @@ __compact_server(void *arg)
 
         WT_ERR(ret);
 
+        ret = (__wt_session_get_dhandle(session, (const char *)uri->data, NULL, NULL, 0));
```

Comment:
Extra brackets on the right side. And if `ret` is something else than `0` and `ENOENT`, we don't capture the failure because of the next call line 429.
Can we add a `WT_ERR(ret)` before line 429?

Reply:
Or even better, use `WT_ERR_ERROR_OK` so we don't need the extra `WT_ERR(ret)`.

Reply:
Nice, WT_ERR_ERROR_OK is way better.